examples/keras_io/tensorflow/vision/deeplabv3_plus.py

```diff
@@ -0,0 +1,326 @@
+"""
+Title: Multiclass semantic segmentation using DeepLabV3+
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Converted to Keras Core: [Muhammad Anas Raza](https://anasrz.com)
+Date created: 2021/08/31
+Last modified: 2023/07/19
+Description: Implement DeepLabV3+ architecture for Multi-class Semantic Segmentation.
+Accelerator: GPU
+"""
+"""
+## Introduction
+
+Semantic segmentation, with the goal to assign semantic labels to every pixel in an image,
+is an essential computer vision task. In this example, we implement
+the **DeepLabV3+** model for multi-class semantic segmentation, a fully-convolutional
+architecture that performs well on semantic segmentation benchmarks.
+
+### References:
+
+- [Encoder-Decoder with Atrous Separable Convolution for Semantic Image Segmentation](https://arxiv.org/pdf/1802.02611.pdf)
+- [Rethinking Atrous Convolution for Semantic Image Segmentation](https://arxiv.org/abs/1706.05587)
+- [DeepLab: Semantic Image Segmentation with Deep Convolutional Nets, Atrous Convolution, and Fully Connected CRFs](https://arxiv.org/abs/1606.00915)
+"""
+
+"""
+## Downloading the data
+
+We will use the [Crowd Instance-level Human Parsing Dataset](https://arxiv.org/abs/1811.12596)
+for training our model. The Crowd Instance-level Human Parsing (CIHP) dataset has 38,280 diverse human images.
+Each image in CIHP is labeled with pixel-wise annotations for 20 categories, as well as instance-level identification.
+This dataset can be used for the "human part segmentation" task.
+"""
+import os
+os.environ['KERAS_BACKEND'] = 'tensorflow'
+
+import keras_core as keras
+from keras_core import layers
+from keras_core import ops
+
+import cv2
+import numpy as np
+from glob import glob
+from scipy.io import loadmat
+import matplotlib.pyplot as plt
+import tensorflow as tf
```

Comment:
You can instead just import `from tensorflow import image as tf_image` and `from tensorflow import data as tf_data`, then move the example to the backend agnostic folder. TF is only needed for the data preprocessing, external to Keras.